core/utils/ethabi.go

```diff
@@ -14,8 +15,23 @@ import (
 )
 
 const (
```

Comment:
Since this is fresh for you, would you mind adding some examples here? They are hard to grok for most people approaching this code, something like:

    input = "0x..."
    output = [0x00, // prefix
                    0x00, // ...

Reply:
Sure, but do you mind if I address this and other concerns in a separate PR? @spooktheducks needs this for Arbitrum to function at Eth Denver, so I'd like to merge ASAP for him.

There are examples in `ethabi_test.go` tests which would be easy to make into golang `func Example`s.

Reply:
Great, I'm going to dismiss your requests for changes, but circle back on them after I've gotten this merged.

Reply:
I moved the tests of the `EthTx` `format` logic into the `adapters` package, to make their relationship to `EthTx` clearer, in `Move EthTx's high-level formatting logic into adapters pkg`. I think those tests constitute adequate examples. LMK if there need to be more.